cleverhans/model.py

```diff
@@ -46,9 +53,9 @@ def get_probs(self, x):
         :return: A symbolic representation of the output probabilities (i.e.,
                 the output values produced by the softmax layer).
         """
-        if 'probs' in self.get_layer_names():
```

Comment:
I guess the fast-path is simpler with the try/catch in this case. The case where probs isn't defined is probably more rare, and this is okay.